### PR TITLE
Add seadata-small and galdata packages

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -20,6 +20,31 @@ unset with_gmp_include with_gmp_lib without_gmp
...
+if [ "$sha256" != "$expected_sha256" ]
+then
+    echo "seadata-small.tgz checksum failure"
+    exit 1;
+fi
+tar -xvf seadata-small.tgz
+
+
+curl -L -O $LINK/galdata.tgz
+sha256=`openssl dgst -sha256 galdata.tgz`
```
Comment (isuruf) [inline recipe/build.sh:39]: @ocefpaf, I ended up checking the sha256 using openssl.

Comment (ocefpaf) [reply recipe/build.sh:39]: :+1:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -31,8 +31,9 @@ requirements:
     - libtool
     - gmp
     - readline 6.2*
-    - perl
+    - perl 5.22.0.1
```
Comment (isuruf) [inline recipe/meta.yaml:34]: @ocefpaf is this the proper way to disable building 3 times?

Comment (ocefpaf) [reply recipe/meta.yaml:34]: For now it is,

**Pull request merged**